LukeZvada/flashcards-server
# Login

#### Changes Made
1. Added file ```def login``` to ```views/auth.py```
1. Modified file ```urls.py``` to include ```login``` functionality.
1. Added ```flashcard.session.sql``` to ```.gitignore```
​
#### Steps to Review
1. Checkout this branch locally.
```
git fetch --all
git checkout ```eh-auth```
```
1.5. Necessary installs if needed
```
pipenv install
python manage.py makemigrations flashcardsapi
python manage.py migrate
python manage.py loaddata users
python manage.py loaddata tokens
```
2. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
3. Test app functionality.
- test on the front end
4. View code file.
- Confirm file modifications are present as indicated above.
- Confirm no unused code or extraneous comments exist.
- Confirm documentation is present where necessary and appropriate.
